### PR TITLE
[Docs] Add a guide on how to deal with file access problems in the wkhtmltopdf

### PR DESCRIPTION
**Pull request by coldic3**
| Q               | A
| --------------- | -----
| Branch?         | 1.10
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | https://github.com/Sylius/RefundPlugin/pull/361, https://github.com/Sylius/InvoicingPlugin/pull/272
| License         | MIT

### CODE REVIEW

File: docs/cookbook/payments/custom-invoice.rst [hunk 1]
```diff
@@ -36,17 +36,31 @@ Example custom configuration:
 
 Make sure to clear the cache each time the configuration is changed.
 
-If you have a permission issue when generating an invoice, you may also need to update the ``config/packages/knp_snappy.yaml`` configuration file:
+.. image:: ../../_images/cookbook/custom-invoice/pdf_with_custom_logo.png
 
-.. code-block:: yaml
+How to add more graphics to the Invoice?
+--------------------------------------------
+In case you would like to add an extra graphic to your Invoice twig template, it is super important to provide access to this file. Let's say you would like to add second image to the Invoice. You may face the problem that ``wkhtmltopdf`` from version 0.16 disables access to the local files by default. Fortunately, there are two options to deal with it:
```
Comment (GSadee) [inline docs/cookbook/payments/custom-invoice.rst:43]: Split this line into several lines and there should be 0.12.6 version I guess 🤔

File: docs/cookbook/payments/custom-invoice.rst [hunk 2]
```diff
@@ -36,17 +36,31 @@ Example custom configuration:
...
 
+    .. code-block:: yaml
 
-.. image:: ../../_images/cookbook/custom-invoice/pdf_with_custom_logo.png
+        knp_snappy:
+            pdf:
+                options:
+                    enable-local-file-access: true
+
+- Specify the exact list of accessible files. As you may have noticed, the logo displays correctly even though local file access is not enabled. This is because we handle it by specifying the exact list of allowed files. The list can be replaced with the ``sylius_refund.pdf_generator.allowed_files`` parameter in the ``config/packages/_sylius.yaml``:
```
Comment (GSadee) [inline docs/cookbook/payments/custom-invoice.rst:54]: Split this line into several lines

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 3]
```diff
@@ -32,18 +32,33 @@ You can achieve that by updating it's path in your ``.env`` file like in example
...
         SYLIUS_REFUND_LOGO_FILE=%kernel.project_dir%/public/assets/custom-logo.png
 
-If you have a permission issue when generating a credit memo, you may also need to update ``config/packages/knp_snappy.yaml`` file with an ``allow`` parameter:
+Make sure to clear the cache each time the configuration is changed.
+
+.. image:: ../../_images/cookbook/custom-credit-memo/customized-pdf.png
+
+How to add more graphics to the Credit Memo?
+--------------------------------------------
+In case you would like to add an extra graphic to your Credit Memo twig template, it is super important to provide access to this file. Let's say you would like to add second image to the Credit Memo. You may face the problem that ``wkhtmltopdf`` from version 0.16 disables access to the local files by default. Fortunately, there are two options to deal with it:
```
Comment (GSadee) [inline docs/cookbook/payments/customizing-credit-memo.rst:41]: Similar as above

File: docs/cookbook/payments/custom-invoice.rst (hunk 1 above)
Comment (coldic3) [reply docs/cookbook/payments/custom-invoice.rst]: Yep, should be 0.12.6 ;d Thanks for catching it.

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
It should be merged only after the next releases of Refund and Invoicing plugins

**Comment by GSadee** [comment]
Thank you, Kevin! :tada:

**Pull request merged**